docs/spec-files/trustping.md

```diff
@@ -54,6 +55,8 @@ creates a `ping` message like this:
 
 **response_requested**: default value is `true`. If false, the `sender` is not requesting a `ping_response` from the `receiver`. If `true`, the `sender` is requesting a response.
 
+**from**: is required if a response is requested so that the receiver knows where to send the response
+
```

Comment:
@brianorwhatever I guess we can remove the `from` after we merged #274

Reply:
thanks @awoie